examples/computer_vision/detr/README.md

```diff
@@ -0,0 +1,33 @@
+# End-to-end Object Detection with Transformers (Carion et al. 2020)
+This example implements the Detection Transformers (DETR) model introduced by Carion et al. (2020) in their [recent paper](https://arxiv.org/abs/2005.12872).  
+DETR drastically simplifies the object detection pipeline by doing away with complex postprocessing steps required by most other architectures to extract bounding boxes.  
+In place of proposals, anchors, and window centers, DETR directly optimizes a set prediction loss to learn where to focus.  
+
+This implementation largely uses the original implementation provided in [this repo](https://github.com/facebookresearch/detr).  With this Determined implementation, you can easily run DETR with distributed training and hyperparameter search by modifying a few flags in the [experiment config](distributed.yaml).
+
+## Files
+This example is structure as follows:
+* [model_def.py](model_def.py): initializes the data, optimizer, and model and specifies the training and evaluation steps.
+* [model.py](model.py): modifies the original [SetCriterion function](https://github.com/facebookresearch/detr/blob/master/models/detr.py#L83) to use horovod allreduce to sync number of bounding boxes.
+* [data.py](data.py): creates a CocoDetection dataset to work with Google Cloud Storage. 
+* [distributed.yaml](distributed.yaml): distributed training experiment to run with COCO dataset on Google Cloud Storage.
+* [const_fake.yaml](const_fake.yaml): single-GPU experiment to run with fake data.
+
+
+## Running the example
+If you have the COCO 2017 dataset available in a Google Cloud Storage bucket, you can modify the `bucket_name` field in [distributed.yaml](distributed.yaml) to point to your location.  Otherwise, you can run this example with fake data using the [const_fake.yaml](const_fake.yaml) config.  Submit the experiment to your Determined cluster by running
```

Comment:
Any chance we can have a `self.data_downloaded` flag set in the `__init__` and download COCO from a URL in Determined way that users don't have to worry about the whole Google Cloud Storage upload if they want to run with real COCO data?

Reply:
added a local backend for data that will download coco from source if directories do not already exist.  coco is fairly large so this will take >5mins.